README.md

```diff
@@ -1,22 +1,42 @@
-# dangermattic
-Shared collection of Danger plugins
+# Dangermattic
+`Dangermattic` builds on [Danger on Ruby](https://danger.systems/ruby/) and is essentially a collection of Danger plugins. Its goal is to provide customisable checks and common utilities to help perform checks on Pull Requests, from simple routine validations to more sophisticated ones.
 
 ## Installation
 
-Add to your `Gemfile`
+Add to your project's `Gemfile`
 ```
-gem 'dangermattic'
+gem 'danger-dangermattic', git: 'https://github.com/Automattic/dangermattic'
 ```
 
-## Usage
+## Available plugins and their usage
```

Comment:
Should we also add a mention that one can check the YARD documentation to learn more about each available plugin and checks exposed by `dangermattic` (i.e. list of plugins, their methods and parameters).

Or do you plan to only add this on a subsequent PR once we have the YARD doc in a good enough place to serve as the proper source of truth?

Also wondering how we plan to update the README as the gem evolves. Will we plan to continue updating this list in the README as we add more checks, how will we make sure those examples are kept up to date if we add/remove/change the checks/plugins we have in the repo, etc 🤔 

I really like having examples directly in the README at least to immediately give the reader an idea of what `dangermattic` can do. But maybe to avoid shooting ourselves in the foot we should name this section `## Example of available plugins and their usage`, to avoid suggesting that this list is exhaustive and instead make it clear that this is just a preview and examples to give an idea, but that to get the full list and documentation of all the available plugins and all their available parameters, one should instead check the (future) YARD doc or the source code? wdyt?

Reply:
Good points, I've been also thinking about that. I see it as just the starting point: eventually, there will be links here to RubyDoc for the plugins documentation.

>  maybe to avoid shooting ourselves in the foot we should name this section `## Example of available plugins and their usage`

I think this is a good compromise! Just making it clear that this isn't exhaustive.